distributor-widget/README.md

```diff
@@ -97,12 +160,3 @@ Mews.Distributor({
 });
 </script>
 ```
-
-## Styles <a id="styles"></a>
-
-Distributor does not use separate CSS files, everything is packed inside the script. For possible customizations, consult [Customization](reference.md) section.
```

Comment:
This doesn't fit anywhere even without referencing the non-existing customization section. So I've removed it. It's less confusing without it. If we want to have something like this, it should be rewritten somehow.